src/app/core/services/common/auth.service.ts

```
@@ -62,6 +81,28 @@
   }
 
+  /**
+   * Update the access token and refresh token in the `user` object
+   * These tokens will be used in all further requests by [JwtInterceptor](../../interceptor/jwt.interceptor.ts)
+   * @param appKey
+   */
+  updateUserTokens(appKey: IntegrationAppKey) {
+    const user: MinimalUser | null = this.userService.getUserProfile();
+    const tokens = this.getTokens(appKey);
+
+    console.log('[x] updated user\'s tokens for', appKey);
```

ESLint

[error] Unexpected console statement.

(no-console)